src/ORM/FieldType/DBYear.php

```diff
@@ -5,12 +5,23 @@
 use SilverStripe\Forms\DropdownField;
 use SilverStripe\Forms\FormField;
 use SilverStripe\ORM\DB;
+use SilverStripe\Model\ModelData;
+use SilverStripe\Core\Validation\FieldValidation\YearFieldValidator;
 
 /**
- * Represents a single year field.
+ * Represents a single year field
  */
 class DBYear extends DBField
 {
+    // MySQL year datatype supports years between 1901 and 2155
+    // https://dev.mysql.com/doc/refman/8.0/en/year.html
+    public const MIN_YEAR = 1901;
+    public const MAX_YEAR = 2155;
```

Comment:
From what I can tell YEAR is pretty much a MySQL only thing

No year type:
https://www.postgresql.org/docs/current/datatype.html
https://sqlite.org/datatype3.html
https://www.sqlservertutorial.net/sql-server-basics/sql-server-data-types/

Reply:
I've added a note to the class doc that this is only intended for MySQL databases

Reply:
I still think `const` is wrong because 
> who knows when mysql may update that in another version, and then we have a const for a value that varies.

but leave as is to avoid ping pong